lib/Backends/Habana/Habana.cpp

```diff
@@ -307,6 +286,8 @@ class TensorHandle final {
   /// Get the underlying data buffer.
   void *getData() const { return buffer_; }
 
+  const std::string &getName() const { return name_; }
```

Comment:
we should add doxygen, but probably in fbcode first.

Reply:
I can add it. This is the source of truth, so good to add it here.